CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [FEATURE] Experimental: Added a new object storage client for OpenStack Swift.
```

Comment:
A couple of minor things:
- Please move the `[FEATURE]` to the group of `[FEATURE]` below
- Remember to add the PR number at the end of the sentence (ie. `#2440`) cause we have a script trying to match PRs with CHANGELOG entries when we release a new Cortex version